notebooks/json_to_txt_conversion.py

```diff
@@ -0,0 +1,31 @@
+"""Converts local JSON Response into a .txt file"""
```

Comment:
I think it's important to include this section of code as examples, however, I would suggest adding some additional information to the docstring explaining how the Google Vision API came into play since it's not part of the main codebase. 

One note for future ideas on this, it may be worth looking into creating "Utility" or "Helper" folder for functions along these lines that help with our processes but are not app centric, rather than keeping them with the notebooks or other parts of the codebase.